src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/PersonAttendingDetailReportITest.java

```diff
@@ -0,0 +1,284 @@
+                                    .id("work_phone")
+                                    .name("Work Phone")
+                                    .dataType("String")
+                                    .headings(null)
+                                    .build(),
+                                StandardReportResponse.TableData.Heading.builder()
+                                    .id("email")
+                                    .name("Email")
+                                    .dataType("String")
+                                    .headings(null)
+                                    .build()))
+                            .build()))
+                    .data(data)
+                    .build())
+            .build();
+    }
+
+    private GroupedReportResponse getTypicalResponse() {
+        return buildResponse(new GroupedTableData()
+                                 .add("415240601", List.of(
+                                     new ReportLinkedMap<String, Object>()
+                                         .add("juror_number", "641500021")
+                                         .add("first_name", "FNAMETWOONE")
+                                         .add("last_name", "LNAMETWOONE")
+                                         .add("juror_postal_address", new ReportLinkedMap<String, Object>()
+                                             .add("juror_address_line_1", "21 STREET NAME")
+                                             .add("juror_address_line_2", "ANYTOWN")
+                                             .add("juror_postcode", "CH1 2AN")),
+                                     new ReportLinkedMap<String, Object>()
+                                         .add("juror_number", "641500011")
+                                         .add("first_name", "FNAMEONEONE")
+                                         .add("last_name", "LNAMEONEONE")
+                                         .add("juror_postal_address", new ReportLinkedMap<String, Object>()
+                                             .add("juror_address_line_1", "11 STREET NAME")
+                                             .add("juror_address_line_2", "ANYTOWN")
+                                             .add("juror_postcode", "CH2 2AB")),
+                                     new ReportLinkedMap<String, Object>()
+                                         .add("juror_number", "641500003")
+                                         .add("first_name", "FNAMETHREE")
+                                         .add("last_name", "LNAMETHREE")
+                                         .add("juror_postal_address", new ReportLinkedMap<String, Object>()
+                                             .add("juror_address_line_1", "3 STREET NAME")
+                                             .add("juror_address_line_2", "ANYTOWN")
+                                             .add("juror_address_line_3", "")
```

Comment:
Should these be null instead of blank (Same for all empty strings)

Reply:
Turns out that no because the fields are emptystring in the SQL file (I used jurors from another test SQL). I figured that was actually a better test to make sure that pulls through, since there's also nulls in those tables.

Reply:
As long as the DB has the empty string and is not null I am happy with this